src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationValidService.java

```diff
@@ -17,7 +17,7 @@
 
 @Service
 public class NotificationValidService {
-    private static final List<NotificationEventType> FALLBACK_LETTER_SUBSCRIPTION_TYPES = Arrays.asList(APPEAL_LODGED, SYA_APPEAL_CREATED_NOTIFICATION);
+    private static final List<NotificationEventType> FALLBACK_LETTER_SUBSCRIPTION_TYPES = Arrays.asList(APPEAL_LODGED, SYA_APPEAL_CREATED_NOTIFICATION, EVIDENCE_RECEIVED_NOTIFICATION);
```

Comment:
According to the ticket, regardless of the subscription for the appellant, therefore this is not a FALLBACK letter, but rather a MANDATORY letter.